src/main-process/main.js

```diff
@@ -61,6 +61,21 @@ app.on('ready', () => {
         'tray-ico': nativeImage.createFromPath(path.join(logosPath, 'museeks-tray.ico'))
     };
 
+    // check if the browser window is offscreen
+    const display = electron.screen.getDisplayNearestPoint(bounds).workArea;
+
+    const onScreen = bounds.x >= display.x
+        && bounds.x + bounds.width <= display.x + display.width
+        && bounds.y >= display.y
+        && bounds.y + bounds.height <= display.y + display.height;
+
+    if(!onScreen) {
+        delete bounds.x;

```

Comment:
Do you mean "overwrite with the x and y bounds" or "overwrite the whole `bounds` object" ?